packages/flutter/lib/src/rendering/proxy_box.dart

```diff
@@ -4072,6 +4074,17 @@ class RenderSemanticsAnnotations extends RenderProxyBox {
     markNeedsSemanticsUpdate();
   }
 
+  /// Adds Semenatics tags to the semantics subtree.
+  Set<SemanticsTag>? get tagsForChildren => _tagsForChildren;
```

Comment:
I wonder if we should simplify the API here and in the Semantics widget to allow only one tag instead of a set. If you need multiple tags (uncommon) you can still wrap your tree in multiple Semantics widgets.